src/map/status.cpp

```diff
@@ -8139,7 +8144,7 @@ static int16 status_calc_aspd(struct block_list *bl, status_change *sc, bool fix
 static int16 status_calc_fix_aspd(struct block_list *bl, status_change *sc, int32 aspd)
 {
 	if (sc == nullptr || sc->empty())
-		return cap_value(aspd, 0, 2000);
+		return cap_value(aspd, 1, 8000);
```

Comment:
Use MIN_ASPD?

Reply:
True, I missed this one here, will fix it.